server/data/incidentTypeConfiguration/questionProgress.ts

```diff
@@ -48,7 +48,7 @@ class ResponseItem {
   get isComplete(): boolean {
     return (
       this.answerConfig &&
-      (!this.answerConfig.commentRequired || this.response.additionalInformation?.length > 0) &&
+      (!this.answerConfig.commentMandatory || this.response.additionalInformation?.length > 0) &&
```

Comment:
Notice how here the `commentMandatory` field is used as this is used to determine if the user completed filling in the questionnaire.

If a comment were to be optional we didn't consider the report "incomplete".

(but please, feel free to double check the logic)